css21/visuren/position-fixed-001.xht

```diff
@@ -0,0 +1,46 @@
+<!DOCTYPE html SYSTEM "about:legacy-compat">
+<html xmlns="http://www.w3.org/1999/xhtml">
+  <head>
+    <title>CSS Test: position:fixed box's position calculated using absolute model</title>
+    <link rel="author" title="Chris Rebert" href="http://chrisrebert.com" />
+    <link rel="help" href="https://drafts.csswg.org/css2/visuren.html#choose-position" />
+    <link rel="help" href="https://drafts.csswg.org/css2/visudet.html#abs-non-replaced-width" />
+    <link rel="help" href="https://drafts.csswg.org/css2/visudet.html#abs-non-replaced-height" />
+    <link rel="match" href="position-fixed-001-ref.xht" />
+    <meta name="flags" content="" />
+    <meta name="assert" content="An element which is position:fixed should have its position calculated correctly using the absolute model when its top/bottom/left/right properties all use the default `auto` value and its parent is position:relative." />
+    <style type="text/css">
+#shifted-column {
+    left: 50%;
+    width: 50%;
+    position: relative;
+    float: left;
+}
+.square {
+  width: 60px;
+  height: 60px;
+}
+#red {
+  background-color: red;
+  position: fixed;
+}
+#green {
+  background-color: green;
+  position: absolute;
+  left: 50%;
+  z-index: 2;
+}
+p {
```

Comment:
> It is recommended to start all tests with the pass-fail-conditions sentence and, since it should not be part of the test itself, then we do not want to style it in any way. Sometimes, this is not possible but, I'd say 99% of the time, it is possible.